bubbly/release_get.go

```diff
@@ -86,7 +92,7 @@ func GetRelease(bCtx *env.BubblyContext) (*builtin.Release, error) {
 	}
 
 	if len(releases.Release) == 0 {
-		return nil, fmt.Errorf("release does not exist")
+		return nil, ErrReleaseNotExist
```

Comment:
I don't think we've been using this error handling/reporting pattern anywhere else? Just checking.